pkg/devfile/adapters/common/utils_test.go

```diff
@@ -128,7 +128,7 @@ func TestGetVolumes(t *testing.T) {
 	}{
 		{
 			name:      "Case 1: Valid devfile with container referencing a volume component",
-			component: []versionsCommon.DevfileComponent{testingutil.GetFakeContainerComponent("comp1"), testingutil.GetFakeVolumeComponent("myvolume1")},
+			component: []versionsCommon.DevfileComponent{testingutil.GetFakeContainerComponent("comp1"), testingutil.GetFakeVolumeComponent("myvolume1", "4Gi")},
```

Comment:
Can we make a const for `4Gi` value so that it can be changed at only one place if need be and avoid any error if more tests are added?